apps/emqx_management/src/emqx_mgmt_api_trace.erl

```diff
@@ -421,6 +436,11 @@ trace(post, #{body := Param}) ->
                 code => 'DUPLICATE_CONDITION',
                 message => ?TO_BIN([Name, " Duplication Condition"])
             }};
+        {error, {bad_type, _}} ->
+            {409, #{
+                code => 'BAD_TYPE',
+                message => <<"Rolling upgrade in progress, create failed">>
```

Comment:
Is the message correct?  It seems it would be something more like "Bad encoding type"?